runtime/src/message_processor.rs

```diff
@@ -118,7 +125,7 @@ impl MessageProcessor {
             );
             timings.accumulate(&invoke_context.timings);
         }
-        Ok(())
+        Ok(ProcessedMessageInfo::default())
```

Comment:
Right now just return a default value. The next PR ([here](https://github.com/solana-labs/solana/pull/21813)) will fill this in with real values.

Reply:
Are you planning to return the updated meter value?

Reply:
Yep! Here's a snippet from the next PR:
https://github.com/solana-labs/solana/blob/3393d2e87130b4c53f396b4cd23841edc52e406a/runtime/src/message_processor.rs#L130-L132